### PR TITLE
Add possibility do not render tiles or to render tiles as a link, remove unneeded code 

### PR DESCRIPTION
**Pull request by manni497**

### CODE REVIEW

File: src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx [hunk 1]
```diff
@@ -12,11 +12,10 @@ export interface PanelSideBarItemProps {
   depth?: number;
   active?: boolean;
   toggledItemIds: string[];
-  toggledSidebar: boolean;
```
Comment (neotob) [inline src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx:15]: this flag wasn't exposed before? or is this a breaking change 💥 ?

File: src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx [hunk 2]
```diff
@@ -75,8 +76,8 @@ export const PanelSideBarLayout = (props: PanelSideBarLayoutProps) => {
         </Nav>
       </nav>
 
-      <section className={classNames({ toggled: !isOpen })}>
-        <PanelSideBar toggledSidebar={isOpen} />
+      <section className={`${renderFirstItemsLevelAsTiles ? "section-tiles" : "section-no-tiles"} ${classNames({ toggled: !isOpen })}`}>
```
Comment (neotob) [inline src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx:79]: couldn't you add the new one also over the classNames function? or doesnt makes sense? 🤔 (to force also future changes more to use the classNames 😄)
https://www.npmjs.com/package/classnames

File: src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx (hunk 1 above)
Comment (neotob) [reply src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx:15]: then perfect and alles gut :D

File: src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx (hunk 2 above)
Comment (manni497) [reply src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx]: @neotob Done with classNames instead, I thought it was better without for the if else, but at the end it's the same.. hope it's fine for you :D

**Review by neotob — APPROVED** [review]

**Pull request merged**